### PR TITLE
Drau/rebrand colors typography

### PR DESCRIPTION
**Pull request by daverau-optimizely**
✅ This is ready for review and merging @jamesopti 

Below is a summary of changes to OUI for the rebrand:

# CSS
- Most objects now have rounded corners (buttons, dropdowns, inner dropdown hovers)
- Arrow-inline (replace Classic triangle with thinner v-shaped arrow from X; Reduce stroke weight to 1px)
- Badge (colors, fonts, padding)
- Blocklist (padding, colors)
- Buttons (all sizes/styles updated, no shadows)
- Button-group only gets outer border radius
- Dropdown (border radius, padded contents)
- Disclose (new arrow style)
- Auto align `input[number]`
- Label no longer bold weight
- Poptip & Popover border radius
- Switch colors (now blue, not green)
- RangeSlider (now blue, not green)
- TabNav (border and text colors)
- Table headers `<th>` font-weight medium (no longer bold)
- Link colors updated to `--light-blue-75`
- `<b>/<strong>` and `weight--bold` now a medium font weight instead of true bold
- `<hr>` included in .oui-rule as a default (hat-tip @zleach)

# React
- Overview story
- Editable input has `cursor--pointer`
- EmptyDashboard colored images (short-term fix)
- RangeSlider set disabled state properly

# Colors
- Fewer shadows, borders
- Updated colors (brand light and dark, good/bad news, admin, grays)
- SASS color variables updated; Added CSS3 variables (Using a mix of CSS3 variables and SASS color variables depending on needs since you can't tint/shade CSS3 vars)
- New tint/shade color mixins in SASS hex colors

# Typography
- New typefaces (Graphik and Lato)
- Host typefaces on S3 bucket tied to CloudFront CDN endpoint
- Reset.css updated to include font-smoothing and text-rendering that changes anti-aliasing defaults (styles moved from X)
- Add .number-display helper for numbers (Lato font utility class)
- Add typography SASS partial (h1-h6, .alpha-.zeta, .milli, .micro, etc)

# Todo (Once HTML changes have landed in Optimizely.git)
- [ ] Take another pass thru OUI CSS and look for clean-up opportunities
- [ ] Remove `HTML FIX` comments
- [ ] Remove quickfix use of !important
- [ ] Clean-up code as needed

### CODE REVIEW

**Comment by daverau-optimizely** [comment]
@jamesopti @kwalker3690 Can I get a review on this? The big changes are listed above. I'm open to feedback, and we have time to incorporate more feedback/long-term changes over the next few weeks as we get into Phase 2 of the rebrand effort.

File: src/components/ArrowsInline/index.scss [hunk 1]
```diff
@@ -9,45 +9,88 @@
 ///   <div><span class="oui-arrow-inline--left"></span> Left</div>
 ////
 
-$border-colored: 4px solid currentColor;
-$border-transparent: 4px solid transparent;
+// $border-colored: 4px solid currentColor;
```
Comment (jamesopti) [inline src/components/ArrowsInline/index.scss]: Nit remove

File: src/components/ArrowsInline/index.scss [hunk 2]
```diff
@@ -9,45 +9,88 @@
...
+  }
+
+  &--small {
+    width: 6px;
+    height: 6px;
+    vertical-align: 20%;
   }
 }
+
+// %#{$namespace}arrow-inline,
```
Comment (jamesopti) [inline src/components/ArrowsInline/index.scss]: remove

File: src/components/Button/index.scss [hunk 3]
```diff
@@ -119,28 +127,28 @@
...
-    }
-
-    &:active {
-      background: map-fetch($button, type, outline-reverse, active, background);
-      border-color: map-fetch($button, type, outline-reverse, active, border);
-      color: map-fetch($button, type, outline-reverse, active, color);
-    }
-  }
-
+  // &--outline-reverse {
```
Comment (jamesopti) [inline src/components/Button/index.scss]: remove

File: src/components/Dropdown/index.scss [hunk 4]
```diff
@@ -156,9 +161,27 @@
...
+      .oui-dropdown--descriptive__content {
+        color: var(--light-blue-75);
+      }
+    }
+  }
+
   &--up {
     bottom: 100%;
-    box-shadow: 0 (-$box-shadow-offset) map-fetch($shadow, light);
+    // box-shadow: 0 (-$box-shadow-offset) map-fetch($shadow, light);
```
Comment (jamesopti) [inline src/components/Dropdown/index.scss]: remove

File: src/oui/partials/elements/_fonts.scss [hunk 5]
```diff
@@ -4,47 +4,168 @@
...
+}
+@font-face {
+  font-family: 'Lato';
+  font-weight: 700;
+  font-style: italic;
+  src: url($font--hosted-url+'/lato/lato-bolditalic.woff2') format('woff2'),
+}
+
+
+// @if variable-exists(include-fonts) {
```
Comment (jamesopti) [inline src/oui/partials/elements/_fonts.scss]: remove

**Comment by jamesopti** [comment]
This looks sane to me. Couple nits about removing commented out code.

File: src/oui/partials/elements/_fonts.scss [hunk 6]
```diff
@@ -4,47 +4,168 @@
...
+    400: regular,
     500: medium,
-    600: sbold,
+    600: semibold,
     700: bold,
-    800: xbold,
-    900: black
   ),
-  url: 'https://dhm5hy2vn8l0l.cloudfront.net/proxima/'
+  // url: 'https://dhm5hy2vn8l0l.cloudfront.net/webfonts/'
```
Comment (kwalker3690) [inline src/oui/partials/elements/_fonts.scss]: Can this be removed

**Pull request merged**